administration/src/shared/components/BlockerDialog.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/destructuring-assignment */
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { Blocker } from 'react-router'
+
+export const BlockerDialog = (p: { title: string; message: string; blocker: Blocker }): ReactElement => {
+  const { t } = useTranslation('misc')
+
+  return (
+    <Dialog
+      open={p.blocker.state === 'blocked'}
+      onClose={p.blocker.reset}
+      aria-labelledby='alert-dialog-title'
+      aria-describedby='alert-dialog-description'>
+      <DialogTitle id='alert-dialog-title'>{p.title}</DialogTitle>
+      <DialogContent>
+        <DialogContentText id='alert-dialog-description'>{p.message}</DialogContentText>
```

Comment:
🔧 Here `<Typography>` component should be used to ensure that we use `body2` here like in other dialogs as body text

Reply:
I set it in the theme to `inherit`.